plugins/always-seekable-body-plugin.rst

```diff
@@ -0,0 +1,25 @@
+Always Seekable Body Plugin
+===========================
+
+The ``AlwaysSeekableBodyPlugin`` ensure that body of the request and body of the response are always seekable.
```

Comment:
ensures that the body of the request and the body of the response...

(i know, lots of "the" but i don't think they can be avoided in english)

maybe: ensure that the request body and the response body are always seekable